README.md

```diff
@@ -1120,13 +1120,14 @@ present script similar to how `$0` works in bash or perl.
 
 `opts` is optional and acts like calling `.options(opts)`.
 
-.version([option], [description], version)
+.version([option], [description], [version])
 ----------------------------------------
 
 Add an option (e.g. `--version`) that displays the version number (given by the
 `version` parameter) and exits the process. If no option is provided, it will default to `--version`.
 If present, the `description` parameter customizes the description of the version
-option in the usage string.
+option in the usage string. You can also pass no `version` parameter (`.version()`),
+in this case, yargs will parse the `package.json` of your module and use its `version` value.
```

Comment:
Maybe we should add that you can also just leave out `description` and/or `option` parameters and still pass a `version` parameter, not only like `.version(null, null, version)` but like `.version(desc, ver)` and `.version(ver)`, too